hedera-node/hedera-app/src/main/java/com/hedera/node/app/info/HederaFileNumbersImpl.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.info;
+
+import com.hedera.node.app.spi.numbers.HederaFileNumbers;
+import com.hedera.node.config.ConfigProvider;
+import com.hedera.node.config.data.FilesConfig;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class HederaFileNumbersImpl implements HederaFileNumbers {
```

Comment:
Why was this class added? It just returns what's already available in `FilesConfig`, right?

Reply:
you are right I fixed it and use the config directly.

Reply:
fixed and committed